lib/interceptor/proxy.js

```diff
@@ -27,36 +27,47 @@ class ProxyAgent extends DispatcherBase {
   constructor (opts) {
     super()
 
-    if (!opts || (typeof opts === 'object' && !(opts instanceof URL) && !opts.uri)) {
+    if (
+      !opts ||
+      (typeof opts === 'object' && !(opts instanceof URL) && !opts.uri)
+    ) {
       throw new InvalidArgumentError('Proxy uri is mandatory')
     }
 
     const { clientFactory = defaultFactory } = opts
     if (typeof clientFactory !== 'function') {
-      throw new InvalidArgumentError('Proxy opts.clientFactory must be a function.')
+      throw new InvalidArgumentError(
+        'Proxy opts.clientFactory must be a function.'
+      )
     }
 
     const url = this.#getUrl(opts)
     const { href, origin, port, protocol, username, password } = url
 
     this[kProxy] = { uri: href, protocol }
     this[kAgent] = new Agent(opts)
-    this[kInterceptors] = opts.interceptors?.ProxyAgent && Array.isArray(opts.interceptors.ProxyAgent)
-      ? opts.interceptors.ProxyAgent
-      : []
+    this[kInterceptors] =
+      opts.interceptors?.ProxyAgent &&
+      Array.isArray(opts.interceptors.ProxyAgent)
+        ? opts.interceptors.ProxyAgent
+        : []
```

Comment:
unrelated change?

Reply:
Yeah, renamed the file and the lint did its thing